Istio supported versions change (#5134)
knative/docs: docs/install/installing-istio.md

## Supported Istio versions

> You can see which version is the latest to have been tested [in the most recent net-istio releases](https://github.com/knative-sandbox/net-istio/releases).
> 

Reviewer: ```suggestion
You can view the latest tested Istio version on the [Knative Net Istio releases page](https://github.com/knative-sandbox/net-istio/releases).
```

suggest something like this for readability
